docs/userguide/distribution.rst

```diff
@@ -161,7 +161,10 @@ without Cython.
 Specifying Your Project's Version
 ---------------------------------
 
-Setuptools can work well with most versioning schemes; there are, however, a
+Setuptools can work well with most versioning schemes. Over the years,
+setuptools has tried to closely follow the 
+`PEP 440 <https://www.python.org/dev/peps/pep-0440/>`_ scheme, but it
+also supports legacy versions. There are, however, a
```

Comment:
Maybe I should rewrite this paragraph, it sounds a bit weird for me since I made the changes.